modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -258,6 +237,28 @@ config TFM_CONNECTION_BASED_SERVICE_API
 
 if TFM_BL2
 
+config TFM_MCUBOOT_SIGNATURE_TYPE
+	string "The signature type used to sign the secure and non-secure firmware images."
+	default "EC-P256"
+	help
+	  Available types: RSA-2048, RSA-3072, EC-P256, EC-P384.
```

Comment:
Should ed25519 be on this list as well?

Reply:
No: https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/105/files#diff-c8402d394cb78c69fcf9bc110193858d9c6beb04277528dc74e30bcfe7a72e5bR50